components/audit_log_filter/log_record_formatter/new.cc

```diff
@@ -49,6 +49,8 @@ AuditRecordString LogRecordFormatterNew::apply(
          << "    <IP>" << make_escaped_string(&audit_record.event->ip) << "</IP>\n"
          << "    <USER>" << make_escaped_string(&audit_record.event->user) << "</USER>\n"
```

Comment:
So for non-JSON formatter we use event->user and for JSON one from extended_info.
I think it is better to be consistent in this...

BTW what about formatter in old.cc ? Is it some legacy we are not supposed to touch? Could you please explain?